core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/pluginloader/Plugin.java

```diff
@@ -105,8 +108,8 @@ public String getUniqueName() {
 	@Override
 	public String getVersion() {
 		if (this.manifest == null) {
-			this.logger.warn("manifest is null, using fakeVersion: {}", this.fakeName);
-			return this.fakeVersion;
+			this.logger.error("Plugin manifest is null, returning \"NaN\" for the version");
+			return ("NaN");
```

Comment:
the version `1.0.0` is Not a Number, so returning `"NaN"` is ambiguous. Better returning empty String or null